en/models/virtual-fields.rst

```diff
@@ -120,6 +120,76 @@ best to define the virtualFields in your model's constructor::
 This will allow your virtualFields to work for any alias you give a
 model.
 
+Pagination and Virtual fields set in controller with JOINS
+=============================================================
+The following example allows to have a counter of a hasMany association and allows to use without problem
```

Comment:
Hi @markstory
What happens I do not see the change in the Doc ..?
I have understood that my example was accepted for the CakePHP Doc.